tests/integration/annotation_import/fixtures/video_annotations.py

```diff
@@ -0,0 +1,49 @@
+import pytest
+from labelbox.data.annotation_types.classification.classification import Checklist, ClassificationAnnotation, ClassificationAnswer, Radio
+from labelbox.data.annotation_types.geometry.point import Point
+from labelbox.data.annotation_types.geometry.rectangle import Rectangle
+
+from labelbox.data.annotation_types.video import VideoObjectAnnotation
+
+
+@pytest.fixture
+def bbox_video_annotation_objects():
```

Comment:
I like the organization of fixtures into a separate folder, but it feels like there could be better organization here.
- Move all fixtures to construct annotation and prediction objects (e.g. `line_inference()`) into this fixture and rename it to `annotation.py`
- Move all data_row fixtures to `fixtures/data_rows.py`
You can feel free to address this in the future, since we should prioritize addressing fixing bugs

Reply:
yes I will address this as I have time, for now renamed this fixture to annotations and will keep in mind your comments